scripting/erm/ERMInterpreter.cpp

```diff
@@ -837,10 +837,10 @@ namespace ERMConverter
 				break;
 			case 'S': //setting variable
 				{
-					if(!trig.params.is_initialized() || trig.params.get().size() != 1)
+					if(!trig.params.size() || trig.params.size() != 1)

```

Comment:
not needed anymore. != 1 is enough

Reply:
And in other places where we check size for exact value

Reply:
Ok